src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import '../../App.scss';
+
+import React from 'react';
+import { Todo, User } from '../../types'; // Import types
+import classNames from 'classnames';
+
+interface TodoInfoProps {
+  todo: Todo;
+  users: User[];
+}
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo, users }) => {
+  const user = users.find(findUser => findUser.id === todo.userId);
```

Comment:
The `find` method is used to locate the user associated with the todo. If no user is found, `user` will be `undefined`, which is handled with optional chaining in the JSX. However, consider providing a fallback or error message if the user is not found to improve the user experience and debugging.

Reply:
i think it can't be not founded